src/components/common/Drawer/Drawer.module.scss

```
@@ -0,0 +1,91 @@
+.drawerContainer {
+  display: none;
+}
+
+@include mobile {
+  .drawerContainer {
+    display: block;
+
+    --transition-speed: 0.3s;
```

sass문법이 더 어울리지 않을까요? 
```suggestion
    $transition-speed: 0.3s;
```

---

수정완료!